lib/suse/toolkit/curlrc_dotfile.rb

```diff
@@ -1,7 +1,12 @@
 require 'etc'
 
-# Yast is not adding proxy credentials to the 'http_proxy' env variable, but writing
-# them to ~/.curlrc. This class is parsing the credentials from there to be used in connection.rb
+# When configuring a proxy with "yast2 proxy", the proxy url and credentials
+# are written to ~/.curlrc and /etc/sysconfig/proxy
```

Comment:
Is the format of `/etc/sysconfig/proxy` the same as curlrc?

Reply:
No, it's in this style: 

```
PROXY_ENABLED="no"
HTTP_PROXY="http://test.url"
```

Reply:
And it seems Yast doesn't write the credentials to `/etc/sysconfig/proxy` see https://bugzilla.suse.com/show_bug.cgi?id=1177232

Reply:
Sooo the code comment is not accurate 😅

Reply:
Improved comment: https://github.com/SUSE/connect/pull/446

Reply:
Although is already merged, just for what I can read I summarize the following (for future us):

- Ruby's Net Module reads `http_proxy` and it's capable of picking up user auth if provided in the `http_proxy` url. Nevertheless is not a common thing to happen, YaST writes the proxy settings to `.curlrc`
- `.curlrc` follows libcurls ["cmdfile" rules](https://ec.haxx.se/cmdline/cmdline-configfile) (quick summary: `--option` or `option` per line)
- `etc/sysconfig/proxy` is written in `ENV` format (`KEY=VALUE`).